Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM centos:7
+FROM centos:8
 
-RUN yum install -y epel-release gcc git krb5-devel python-devel python2 python3
-RUN yum install -y python-pip
-RUN pip install --upgrade pip ; pip install --upgrade setuptools
-RUN pip install koji
-RUN pip3 install tox twine setuptools wheel codecov
-RUN curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 --output jq \
-    && chmod +x jq && mv jq /usr/local/bin/jq
+RUN yum install -y epel-release python2 python36
+RUN yum install -y gcc git jq krb5-devel openssl-devel libcurl-devel rpm-devel \
+    python{2,3} python{2,3}-{devel,pip}
```

Comment:
For this particular image we are fine, because `centos:8` symlinks `/bin/sh` to `bash`.
But is good to keep in mind that those bash expansions might not work if we ever switch the `FROM` image with something that ships the real `/bin/sh` (`RUN` runs stuff with `/bin/sh`)

Reply:
Good reminder but I don't think it is a problem since we have to pass the docker build :)